src/lib/arrays.js

```diff
@@ -0,0 +1,111 @@
+const getNthElement = (index, array) => {
```

Comment:
You should remove all the comments in this file, they are just adding noise and dont help with anything really